packages/mobile/src/screens/root-screen/RootScreen.tsx

```diff
@@ -77,6 +83,14 @@ export const RootScreen = ({
     }
   }, [dispatch, isLoaded, accountStatus])
 
+  // Reset the player when the app is loaded for the first time. Fixes an issue
+  // where after a crash, the player would persist the previous state. PAY-1412.
+  useEffect(() => {
+    if (isLoaded) {
+      dispatch(playerActions.reset({ shouldAutoplay: false }))
+    }
+  }, [isLoaded, dispatch])
+
```

Comment:
Could we just put this dispatch on L68 instead of having a new useEffect?

Reply:
I think we want to keep them separate bc we don't want to clear this state if a user was not signed in, then signs in? L68 runs when the user has successfully signed in only. We want this to run when the app loads for the first time regardless of sign-in state.

Reply:
I guess could add `if (isLoaded)`-only check to the above useEffect. I could go either way lmk.

Reply:
chatted with @piazzatron think he's right!